Makefile

```diff
@@ -71,7 +69,7 @@ push-cli: ## Build and publish CLI.
 	gsutil cp -r ./cli/bin/* gs://${BUCKET_PATH}/
 	gsutil setmeta -r -h "Cache-Control:public, max-age=30" gs://${BUCKET_PATH}/\*
 
-clean: clean-ui clean-agent clean-cli clean-docker clean-extensions ## Clean all build artifacts.
```

Comment:
i see you removed the docker debug, so we need 
1. to clean the make file from its reference like: "push-docker-debug"
2. remove build-push-featurebranch-debug.sh

Reply:
@IgorGov I have actually removed `devops/build-push-featurebranch-debug.sh` as well but I forgot to remove the `push-docker-debug` rule. Done with https://github.com/up9inc/mizu/commit/9b2f01a84237bd7f53d0d893c155ed493819a416